docs/blog/2019-08-30-speed-up-your-time-to-interactive-by-delaying-third-party-scripts/index.md

```diff
@@ -28,7 +28,7 @@ Harry from [marketingexamples.com](https://marketingexamples.com/) linked me to
 
 When looking at the performance data, the site's first meaningful paint took slightly over a second, but the time to interactive was _eleven seconds_. Yikes!
 
-The reason for this was that I recently switched to [Segment](https://segment.com) (using [gatsby-plugin-segment-js](//packages/gatsby-plugin-segment-js/)) and was loading other scripts through that, like support chat and analytics. These scripts all counted towards my "time to interactive".
+The reason for this was that I recently switched to [Segment](https://segment.com) (using [gatsby-plugin-segment-js](https://www.gatsbyjs.org/packages/gatsby-plugin-segment-js/)) and was loading other scripts through that, like support chat and analytics. These scripts all counted towards my "time to interactive".
```

Comment:
no domain on internal links - it's enough to remove the double slash

```suggestion
The reason for this was that I recently switched to [Segment](https://segment.com) (using [gatsby-plugin-segment-js](/packages/gatsby-plugin-segment-js/)) and was loading other scripts through that, like support chat and analytics. These scripts all counted towards my "time to interactive".
```

Reply:
Agreed, let's commit this suggestion and get this in